components/wordPrompt.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.nextWords.length != 0) {
```

Comment:
If you follow my other comment, this if statement/branching shouldn't be necessary